### PR TITLE
Fix: Metadata filters doesn't seem to work for Qdrant

### PR DESCRIPTION
**Pull request by shodevacc**
While using Qdrant as a vectorstore in a query engine the filters didn't work. This is a fix for the same.

### CODE REVIEW

File: examples/qdrantdb/preFilters.ts [hunk 1]
```diff
@@ -0,0 +1,61 @@
+import * as dotenv from "dotenv";
```
Comment (marcusschiesser) [inline examples/qdrantdb/preFilters.ts:1]: example looks great, thanks!

File: packages/core/src/storage/vectorStore/QdrantVectorStore.ts [hunk 2]
```diff
@@ -289,7 +289,7 @@ export class QdrantVectorStore implements VectorStore {
    * @param query The VectorStoreQuery to be used
    */
   private async buildQueryFilter(query: VectorStoreQuery) {
-    if (!query.docIds && !query.queryStr) {
+    if (!query.docIds && !query.queryStr && !query.filters) {
```
Comment (marcusschiesser) [inline packages/core/src/storage/vectorStore/QdrantVectorStore.ts:292]: Did you test this without `filters` being specified?

File: examples/qdrantdb/README.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+
+How to run `examples/qdrantdb/preFilters.ts`:
+
+Add your OpenAI API Key into a file called `.env` in the parent folder of this directory. It should look like this:
+
+```
+OPEN_API_KEY=sk-you-key
+```
+
+Now, open a new terminal window and inside `examples/qdrantdb`, run `npx ts-node preFilters.ts`.
```
[the remark this answers is not in the document]
Comment (shodevacc) [reply examples/qdrantdb/README.md:11]: Thanks will do.

File: packages/core/src/storage/vectorStore/QdrantVectorStore.ts (hunk 2 above)
Comment (shodevacc) [reply packages/core/src/storage/vectorStore/QdrantVectorStore.ts:292]: Yes, it does work. I can update the example to showcase that as well if needed

Comment (marcusschiesser) [reply packages/core/src/storage/vectorStore/QdrantVectorStore.ts:292]: If you have time, that would be great of course!

**Comment by marcusschiesser** [comment]
@shodevacc this fixes https://github.com/run-llama/LlamaIndexTS/issues/622, right?

**Comment by shodevacc** [comment]
Yes this would fix #622

File: packages/core/src/storage/vectorStore/QdrantVectorStore.ts (hunk 2 above)
Comment (shodevacc) [reply packages/core/src/storage/vectorStore/QdrantVectorStore.ts:292]: Updated in the new commit

File: examples/qdrantdb/README.md [hunk 4]
```diff
@@ -0,0 +1,11 @@
+# Qdrant Vector Store Example
```
[the remark this answers is not in the document]
Comment (marcusschiesser) [reply examples/qdrantdb/README.md:1]: @shodevacc please have a look into this, then we could merge the PR

File: .changeset/clever-doors-invite.md
Comment (EmanuelCampos) [inline .changeset/clever-doors-invite.md:1]: why deleting changeset?

File: .changeset/eight-dogs-burn.md
Comment (EmanuelCampos) [inline .changeset/eight-dogs-burn.md:1]: why deleting changeset?

File: .changeset/strange-waves-sit.md
Comment (EmanuelCampos) [inline .changeset/strange-waves-sit.md:1]: why deleting changeset?

File: .changeset/grumpy-meals-argue.md
Comment (EmanuelCampos) [inline .changeset/grumpy-meals-argue.md:1]: we probably dont need 2 changeset

File: .changeset/happy-bikes-kiss.md
Comment (EmanuelCampos) [inline .changeset/happy-bikes-kiss.md:1]: we probably dont need 2 changeset

**Review by EmanuelCampos — COMMENTED** [review]
there's a few nits and had some updates on create-llama and other packages, can you check this?

File: examples/qdrantdb/README.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (shodevacc) [reply examples/qdrantdb/README.md:1]: Sorry about the delay. I cleaned up the commits. It should be fine now as I removed the releasing packages commit.

**Review by marcusschiesser — APPROVED** [review]
Thanks @shodevacc

**Pull request merged**